### PR TITLE
Update Digital Subscriptions Banner Copy

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Updates Digital Subscriptions Banner Copy as requested here: https://trello.com/c/XBEWOYr4/3103-urgent-ds-banner-headline-and-standfirst-copy-tweaks-to-support-the-brand-moment-launch-friday-12-june

## Does this change need to be reproduced in dotcom-rendering ?

N/A

## Mobile
Screenshot 2020-06-11 at 12 46 47

## Tablet
Screenshot 2020-06-11 at 12 46 28

## Desktop
Screenshot 2020-06-11 at 12 47 06

### CODE REVIEW

**Comment by lucymonie** [comment]
This is probably more of a call for Cloe to make, but the line break in the heading looks a bit rough on mobile to me.

**Review by lucymonie — APPROVED** [review]

**Comment by GHaberis** [comment]
@lucymonie yeah I'm waiting on cloe's feedback before merging this 👍

**Comment by GHaberis** [comment]
Latest mobile layout looks like...

Screenshot 2020-06-11 at 15 33 04

**Pull request merged**
